pkg/reconciler/v1alpha1/eventlistener/eventlistener.go

```diff
@@ -565,13 +565,13 @@ func getContainer(el *v1alpha1.EventListener) corev1.Container {
 			"-el-name", el.Name,
 			"-el-namespace", el.Namespace,
 			"-port", strconv.Itoa(*ElPort),
+			"-tls-cert", elCert,
```

Comment:
this looks like we just changed the position of the args...did that actually solve the issue? 🤯

Reply:
I am also confused about this.

Reply:
Issue is whatever args we set after `-is-multi-ns` the args value for those are not effected in initializtion.go file

Thats why just moved those args above `-is-multi-ns`

Reply:
Not sure what's the issue with `-is-multi-ns` :thinking:

Reply:
got it, I think setting `is-multi-ns`: "false" is messing things up. Maybe we can just not set the flag when its false and then just set `is-multi-ns` when we need to set it to true.

Reply:
From https://golang.org/pkg/flag/#hdr-Command_line_flag_syntax:

```
The following forms are permitted:

-flag
-flag=x
-flag x  // non-boolean flags only

```

In the deployment that we generate, we are using the last form (-flag x) which does not work for boolean flags.

Reply:
Yes will modify logic for `is-multi-ns`

Reply:
@dibyom That was not the issue. Issue is something else. Even if we remove -is-multi-ns, I found that certain permutations weren't working. So I decided to change the way we pass Arg and every permutations work.

https://github.com/tektoncd/triggers/pull/838/files

Reply:
With that changes it fixes issue with `--is-multi-ns` but still we get permission issue
```
E1123 08:17:43.821688       1 reflector.go:178] k8s.io/client-go@v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible/tools/cache/reflector.go:125: Failed to list *v1alpha1.ClusterTriggerBinding: clustertriggerbindings.triggers.tekton.dev is forbidden: User "system:serviceaccount:tlstest:tekton-triggers-tls-sa" cannot list resource "clustertriggerbindings" in API group "triggers.tekton.dev" at the cluster scope
E1123 08:17:47.527568       1 reflector.go:178] k8s.io/client-go@v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible/tools/cache/reflector.go:125: Failed to list *v1alpha1.ClusterTriggerBinding: clustertriggerbindings.triggers.tekton.dev is forbidden: User "system:serviceaccount:tlstest:tekton-triggers-tls-sa" cannot list resource "clustertriggerbindings" in API group "triggers.tekton.dev" at the cluster scope
E1123 08:17:54.554289       1 reflector.go:178] k8s.io/client-go@v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible/tools/cache/reflector.go:125: Failed to list *v1alpha1.ClusterTriggerBinding: clustertriggerbindings.triggers.tekton.dev is forbidden: User "system:serviceaccount:tlstest:tekton-triggers-tls-sa" cannot list resource "clustertriggerbindings" in API group "triggers.tekton.dev" at the cluster scope

```

Reply:
I will be closing this as its fixed by  [838](https://github.com/tektoncd/triggers/pull/838) PR